infra/blueprint-test/test/tfvet_test.go

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package test
+
+ import (
+	 "bytes"
+	 "log"
+	 "os"
+	 "path"
+	 "testing"
+
+	 "github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	 testingiface "github.com/mitchellh/go-testing-interface"
+	 "github.com/stretchr/testify/assert"
+ )
+
+ func TestTerraformVet(t *testing.T) {
+	 cwd, err := os.Getwd()
+	 if err != nil {
+		 t.Fatalf("unable to get wd :%v", err)
+	 }
+	 libraryPath := path.Join(cwd, "../examples/policy-library")
+
+	 t.Run("Valid configuration", func(t *testing.T) {
```

Comment:
Can't these just be a table test? Thinking something like
```go
{
service : "cloudresourcemanager.googleapis.com",
},
{
service: "oslogin.googleapis.com",
errMsgContains: "GCPServiceUsageConstraintV1"
}
```

Reply:
added table test

```
		{
			name:    "Valid configuration",
			service: "cloudresourcemanager.googleapis.com",
		},
		{
			name:           "Configuration with violations",
			service:        "oslogin.googleapis.com",
			errMsgContains: "GCPServiceUsageConstraintV1",
		},
```